src/actions/account.js

```diff
@@ -274,9 +243,13 @@ export const updateAccountDelegateStats = account =>
       token, networkConfig, address, limit: 1, type: transactionTypes.registerDelegate,
     });
     const block = await getBlocks(liskAPIClient, { generatorPublicKey: publicKey, limit: 1 });
-    dispatch(delegateStatsLoaded({
-      lastBlock: (block.data[0] && block.data[0].timestamp) || '-',
-      txDelegateRegister: transaction.data[0],
+    dispatch(accountUpdated({
+      token: 'LSK',
```

Comment:
this method should update the balance of the active token, not only 'LSK'

Reply:
This method is `updateAccountDelegateStats`. It doesn't update balance, but delegate stats, which exist only for LSK.

Reply:
I changed it to use the token variable that is already defined in the action. https://github.com/LiskHQ/lisk-hub/pull/2003/commits/6c323489a54f00d2dc63833637171341489067da